studiopress/atomic-blocks
# Prevent Chromium from always downloading

* It looks like [@wordpress/scripts@9.0.0](https://github.com/WordPress/gutenberg/blob/b83f94845501378dfd78f1a10db785e05dee2d87/packages/scripts/CHANGELOG.md#900-2020-04-30) now defers downloading Chromium
until `test-e2e` runs

* Also, this uses `npm:puppeteer-core@3.0.0`, like [in Gutenberg](https://github.com/WordPress/gutenberg/commit/8afed75c6f2daa2624b0f91781aa4bee163d4e93#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R164)
### How to test

1. `rm -rf node_modules`
2. `npm install`
3. Expected: Chromium is not downloaded
4. `npm run build && npm run env start && npm run test:e2e`
5. Expected: Chromium is then downloaded:
<img width="446" alt="chromium-downloaded-here" src="https://user-images.githubusercontent.com/4063887/84696773-9ad2cd80-af12-11ea-9349-011c8cc46544.png">

### Documentation
* Probably no documentation required. This should only affect the developer experience of this plugin.

### Suggested changelog entry
- Prevent Chromium from downloading on `npm install`.

### Notes

* CSS linting failed with the new `@wordpress/scripts@11.0.0` package. If you like this overall approach, we can talk about whether to make the CSS conform to the new ruleset, or change the ruleset.
* The CSS linting also failed with `@wordpress/scripts@9.0.0`, so I thought we might as well use the more recent `11.0.0`. But your call 😄
